style.css

```diff
@@ -89,13 +89,429 @@ h1 {
 
 .image22 {
   width: 375px;
-  position: absolute;
-  margin-top: 1.5rem;
+  margin-top: 1.3rem;
   left: 0;
 }
 
 .image2 {
-  width: 375px;
+  width: 320px;
+}
+
+.projects {
+  display: grid;
+}
+
+.projects-title {
+  width: 181px;
+  height: 50px;
+  margin-left: 1.3rem;
+}
+
+.projects-title > h2 {
+  font-family: "Playfair Display", serif;
+  font-size: 50px;
+  font-weight: 400;
+  line-height: 100%;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.projects-1 {
+  margin-top: 4rem;
+}
+
+.projects-1_img {
+  width: 320px;
+  height: 186px;
+}
+
+.project-1_name {
+  display: grid;
```

Comment:
- [ ] Kindly, Try to remove the right margin on your project and about sections to replicate the Figma design and also for the best user experience 👍 

![image](https://user-images.githubusercontent.com/102757126/219679892-2104a24c-ed3c-43ef-9e97-17e387326286.png)